File: JS/8kyuOppositesAttract.js
//8kyu Opposites Attract
lovefunc = (flower1, flower2) => {
    if (flower1 % 2 === 0 && flower2 % 2 != 0){
        return true
    } else if 
        (flower1 % 2 != 0 && flower2 % 2 === 0) {
          return true
    } else if
        (flower1 === 0 && flower2 % 2 != 0) {
          return true
    } else if
        (flower1 === 0 && flower2 === 0) {
          return false
    } else {
          return false
    }
  }
//test lovefunc(1,4) then lovefunc(0,0)

File: C#/8kyuOppositesAttract.cs
using System;

public class LoveDetector
{
    public static bool lovefunc(int flower1, int flower2)
    { 
      int f1 = flower1 % 2;
      int f2 = flower2 % 2;
      if (f1 == 0 && f2 == 1) {
        return true;
      } else if (f1 == 1 && f2 == 0) {
        return true;
      } else {
        return false;
      }
     }
}